java/ingest/ingest-batcher-submitterv2/src/test/java/sleeper/ingest/batcher/submitterv2/IngestBatcherSubmitterLambdaIT.java

```diff
@@ -74,15 +81,53 @@ void shouldStoreFileIngestRequestFromJson() {
                     "}";
 
             // When
-            lambda.handleMessage(json, RECEIVED_TIME);
+            lambda().handleMessage(json, RECEIVED_TIME);
 
             // Then
-            assertThat(store.getAllFilesNewestFirst())
+            assertThat(batcherStore().getAllFilesNewestFirst())
                     .containsExactly(
                             fileRequest(testBucket + "/test-file-1.parquet"));
             assertThat(receiveDeadLetters()).isEmpty();
         }
 
+        @Test
+        void shouldStoreFileFindFileEvenWithoutFileExtension() {
```

Comment:
This test name reads strangely. We could make it clear from the names which is which between this test and the next.